configs/peer/config.json

```diff
@@ -25,9 +24,9 @@
     "GOSSIP_PERIOD_MS": 1000
   },
   "TORII": {
-    "P2P_ADDR": null,
-    "API_URL": null,
-    "TELEMETRY_URL": null,
+    "P2P_ADDR": "http://127.0.0.1:1337",
+    "API_URL": "http://127.0.0.1:8080",
+    "TELEMETRY_URL": "http://127.0.0.1:8180",
```

Comment:
What about addressed? Do you think we need to keep them here? Or maybe better to keep them as nulls?

Reply:
Yeah, lets keep them `null`, since this addresses would be replaced anyway to setup network